scripts/logrotate/freeradius

```diff
@@ -4,45 +4,32 @@
 #  You can use this to rotate the /var/log/radius/* files, simply copy it to /etc/logrotate.d/radiusd
 #
 
-#
-#    Global options for all files
-#
-daily
-rotate 14
-missingok
-compress
-delaycompress
-notifempty
-su radiusd radiusd
-
 #
 #  The main server log
 #
 /var/log/radius/radius.log {
 	copytruncate
+	daily
+	rotate 14
+	missingok
+	compress
+	delaycompress
+	notifempty
+	su radiusd radiusd
 }
 
 #
 #  Session monitoring utilities
 #
-/var/log/radius/checkrad.log /var/log/radius/radwatch.log {
-	nocreate
-}
-
+/var/log/radius/checkrad.log /var/log/radius/radwatch.log
 #
-#  Session database modules
+#  SQL log files
 #
-/var/log/radius/radutmp /var/log/radius/radwtmp {
-	nocreate
-}
-
+/var/log/radius/sqllog.sql
```

Comment:
Why were these options deleted?  There's no need for that as part fo a commit which cleans up the global config file.

Reply:
The new config file should be equivalent to the old one. I just wanted to avoid redundant configuration as far as possible. For example:

```
/var/log/radius/sqllog.sql                                                        
#
# some comments                                                                                 
#
/var/log/radius/radacct/*/detail {                                                
        create                                                                    
        daily                                                                     
        rotate 14                                                                 
        missingok                                                                 
        compress                                                                  
        delaycompress                                                             
        notifempty                                                                
        su radiusd radiusd                                                        
}                              
```

This means that the options block `{ create ... su radiusd radiusd }` applies to both, `/var/log/radius/sqllog.sql` and `/var/log/radius/radacct/*/detail`.